include/abb_librws/rws_client.h

```diff
@@ -625,7 +653,7 @@ class RWSClient : public POCOClient
   /**
    * \brief Static constant for the log's size.
    */
-  static const size_t LOG_SIZE = 10;
+  static const size_t LOG_SIZE = 20;
```

Comment:
Just curious, but why the 100% increase?

Reply:
The log filled up too quickly with only 10, and 20 seemed reasonable after some testing